cxbox-core/src/main/java/org/cxbox/core/service/rowmeta/RowResponseService.java

```diff
@@ -112,8 +115,15 @@ public MetaDTO getResponse(RowMetaType type, DataResponseDTO dataDTO, BusinessCo
 			Class<? extends FieldMetaBuilder> fieldMetaBuilder) {
 		EngineFieldsMeta fieldsNode = getMeta(bc, type, dataDTO, true);
 		if (linkedDictionaryService != null) {
-			linkedDictionaryService.fillRowMetaWithLinkedDictionaries(fieldsNode, bc, dataDTO, type == RowMetaType.META_EMPTY);
+				linkedDictionaryService.fillRowMetaWithLinkedDictionaries(fieldsNode, bc, dataDTO, type == RowMetaType.META_EMPTY);
+		}
+
+		//add changedNowParam in parameter RowDependentFieldsMeta<T> fields for FieldMetaBuilder
```

Comment:
why not in public EngineFieldsMeta getMeta(BcIdentifier bc, RowMetaType type, DataResponseDTO dataDto, boolean visibleOnly) ??

add comment why we need to add this field with separate code (not as other dto fields)